src/Lykke.Service.Zcash.Api.AzureRepositories/Addresses/AddressEntity.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Lykke.AzureStorage.Tables;
+using Lykke.Service.Zcash.Api.Core.Domain;
+using Lykke.Service.Zcash.Api.Core.Domain.Addresses;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Lykke.Service.Zcash.Api.AzureRepositories.Addresses
+{
+    public class AddressEntity : AzureTableEntity, IAddress
+    {
+        public AddressEntity()
+        {
+        }
+
+        public AddressEntity(string partitionKey, string rowKey)
+        {
+            PartitionKey = partitionKey;
+            RowKey = rowKey;
+        }
+
+        [IgnoreProperty]
+        public ObservationCategory ObservationSubject
+        {
+            get => (ObservationCategory)Enum.Parse(typeof(ObservationCategory), PartitionKey);
```

Comment:
We will get only three partitions, with this aprocah. Probably, we can use combnation of observation and category hash of address as a partition key?

Reply:
Yep, three partitions only. That means up to active clients count of rows per partition. I don't know if it's crucial for Azure Table Storage, but now it's more convenient to get addresses by single category.

Reply:
Refactored address storage to achieve better granularity.
Balance => PK: address
History => PK: address, RK: From|To